packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -492,7 +502,8 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     assign(buttonProps, {
       onClick: undefined,
       tabIndex: -1,
-      'data-is-focusable': false
+      'data-is-focusable': false,
+      ...primaryActionButtonProps
```

Comment:
This will still make the props go to the container since the container props are not defined until line `514`. You should do something like

```ts
assign(buttonProps, {...primaryActionButtonProps});
```

after line `containerProps` have been determined.

Reply:
Ok, good catch.